admin_guide/multiproject_quota.adoc

```diff
@@ -13,17 +13,25 @@ toc::[]
 
 == Overview
 
-A multi-project quota, defined by a `*ClusterResourceQuota*` object, allows xref:../admin_guide/quota.adoc#admin-guide-quota[quota]
-to be shared across multiple projects.  Resources used in each selected project
-will be aggregated and that aggregate will be used to limit resources across
-all the selected projects.
+A multi-project quota, defined by a `*ClusterResourceQuota*` object, allows
+xref:../admin_guide/quota.adoc#admin-guide-quota[quota] to be shared across
```

Comment:
I believe the plural for quota is quotas, so either:

s/quota/a quota or s/quota/quotas

That does depend on if you can share one quota object over multiple projects, I guess...